// local_vlm_server.proto
syntax = "proto3";

package local_vlm_server;

// 空のメッセージ型

// URL情報を含むメッセージ
message SendImageRequest {
  repeated string images = 1;
  optional string prompt = 2;
}
message SendImageReply {
 string response = 1;
}

service LocalVlmServerService {
  // 新しいURLを送信
  rpc SendImage (SendImageRequest) returns (SendImageReply) {}
}

